public/app/features/dashboard-scene/saving/DashboardPrompt.tsx

```diff
@@ -178,10 +187,28 @@ export function ignoreChanges(scene: DashboardScene | null) {
     return true;
   }
 
+  // Ignore changes if the original is empty (new dashboard)
+  if (original && 'schemaVersion' in original && isEmptyDashboard(original)) {
+    return true;
+  }
+
   const { canSave, fromScript, fromFile } = scene.state.meta;
   if (!contextSrv.isEditor && !canSave) {
     return true;
   }
 
   return !canSave || fromScript || fromFile;
 }
+
+function isEmptyDashboard(dashboard: Dashboard): boolean {
+  if (!dashboard) {
+    return true;
+  }
+
+  const hasNoPanels = !dashboard.panels?.length;
```

Comment:
🤔 can we double check if `DashboardPrompt` is only being used for v1 dashboards? if we are using them in v2, we will need to check different properties depending on the schema